html/semantics/forms/constraints/META.yml

```
@@ -27,15 +26,13 @@ links:
         - test: infinite_backtracking.html
         - test: inputwillvalidate.html
         - test: form-validation-willValidate-datalist.html
-        - test: form-validation-reportValidity.html
         - test: form-validation-validity-badInput.html
         - test: form-validation-validity-rangeOverflow.html
         - test: form-validation-validity-tooShort.html
         - test: form-validation-validity-valueMissing.html
         - test: input-pattern-dynamic-value.html
         - test: radio-valueMissing.html
-        - test: form-validation-validity-customError.html
-        - test: form-validation-validity-stepMismatch.html
+        - test: form-validation-validity-customError.html        
```

Trailing whitespace makes it look like a change in the diff:

```suggestion
        - test: form-validation-validity-customError.html
```